twitcheventsub/src/modules/eventsub.rs

```diff
@@ -177,15 +176,15 @@ pub fn events(
       NetworkMessage::Text(msg) => {
         #[cfg(feature = "only_raw_responses")]
         {
-          let _ = message_sender.send(ResponseType::RawResponse(msg.clone()));
+          let _ = message_sender.send(ResponseType::RawResponse(msg.to_string()));
         }
 
-        let message = serde_json::from_str(&msg);
+        let message = serde_json::from_str(msg.as_str());
```

Comment:
so these may seem like useless changes but the data type of NetworkMessage::Text has changed. The data is now of the type [`Utf8Bytes`](https://docs.rs/tungstenite/latest/tungstenite/protocol/frame/struct.Utf8Bytes.html) which only has a as_str function and not a to_string function. in this case it makes sense to directly call as_str as the function needs &str. in the other cases as the function specifically needs a String, it makes sense to call to_string(). This should explain the changes in other locations where ever `NetworkMessage::Text` is used, the changes to the internal `IRCMessage` allow us to skip a unnecessary `to_owned`/`to_string`/`into`